packages/@aws-cdk/aws-appmesh/test/test.virtual-gateway.ts

```diff
@@ -429,6 +630,102 @@ export = {
 
       test.done();
     },
+
+    'with client\'s TLS certificate from SDS': {
```

Comment:
Let's just use double quotes for strings that include `'`:

```suggestion
    "with client's TLS certificate from SDS": {
```

Reply:
Well noted! This is great to know.